library/cwm/src/lib/cwm/widget.rb

```diff
@@ -602,13 +602,31 @@ class RadioButtons < AbstractWidget
     include ItemsSelection
     abstract_method :label
 
+    # @!method vspacing
+    #   @return [Fixnum] space between the options
```

Comment:
in fact radio buttons support both vspacing and hspacing. So I think it make sense to add both.

see https://github.com/yast/yast-yast2/blob/master/library/cwm/src/modules/CWM.rb#L594